src/main/java/uk/gov/hmcts/ccd/domain/service/casedeletion/TimeToLiveService.java

```diff
@@ -118,50 +117,71 @@ public void verifyTTLContentNotChanged(Map<String, JsonNode> expected, Map<Strin
+        if (isTtlCaseFieldPresent(updatedCaseData) && isTtlCaseFieldPresent(currentCaseData)) {
+            updatedTTL = getTTLFromJson(updatedCaseData.get(TTL_CASE_FIELD_ID));
+            currentTTL = getTTLFromJson(currentCaseData.get(TTL_CASE_FIELD_ID));
         }
 
-        if (beforeCallbackTTL == null || currentTTLInDatabase == null) {
+        if (updatedTTL == null || currentTTL == null) {
+            // if `updatedTTL` is null then all is OK as no TTL in place, i.e. no deletion will occur.
+            // if `currentTTL` is null then all is OK as no previous suspension to validate against.
             return;
         }
 
-        LocalDate localDate = LocalDate.now().plusDays(applicationParams.getTtlGuard());
+        // checking if `TTL.suspended` has changed value
+        if (updatedTTL.isSuspended() != currentTTL.isSuspended()
+            || overrideTTLIsChanged(currentTTL.getOverrideTTL(), updatedTTL.getOverrideTTL())) {
+            LocalDate ttlGuardDate = LocalDate.now().plusDays(applicationParams.getTtlGuard());
+            LocalDate resolvedTTL = getResolvedTTL(updatedTTL);
 
-        // checking if TTL.suspended has changed value
-        if ((beforeCallbackTTL.isSuspended() != currentTTLInDatabase.isSuspended())
-            && (!beforeCallbackTTL.isSuspended()
-            && (beforeCallbackTTL.getSystemTTL() != null
-            && beforeCallbackTTL.getSystemTTL().isBefore(localDate)))
-            && beforeCallbackTTL.getOverrideTTL() != null
-            && beforeCallbackTTL.getOverrideTTL().isBefore(localDate)) {
-            throw new ValidationException(TIME_TO_LIVE_SUSPENSION_ERROR_MESSAGE);
+            // NB: if `resolvedTTL` is non-null then no suspension in place  ...
+            //     ... so given we know `TTL.suspended` has changed value it must have been lifted
+
+            // validate: Unsetting a suspension can only be allowed if the deletion will occur beyond the guard period
+            if (resolvedTTL != null && resolvedTTL.isBefore(ttlGuardDate)) {
+                throw new ValidationException(TIME_TO_LIVE_GUARD_ERROR_MESSAGE);
+            }
         }
     }
 
+    // check if overrideTTL has changed value
+    private boolean overrideTTLIsChanged(LocalDate currentOverrideTTL, LocalDate updatedOverrideTTL) {
+        if (currentOverrideTTL == null && updatedOverrideTTL == null
+            || (currentOverrideTTL != null && updatedOverrideTTL != null
+            && currentOverrideTTL.equals(updatedOverrideTTL))) {
+            return false;
+        }
+        return true;
+    }
+
     public LocalDate getUpdatedResolvedTTL(Map<String, JsonNode> caseData) {
-        LocalDate resolveTTL = null;
-        TTL afterCallbackTTL = null;
-        if (caseData.get(TTL_CASE_FIELD_ID) != null) {
-            afterCallbackTTL = getTTLFromJson(caseData.get(TTL_CASE_FIELD_ID));
+        TTL ttl = null;
+
+        if (isTtlCaseFieldPresent(caseData)) {
```

Comment:
It looks like `isTtlCaseFieldPresent` is always called before a call to `getTTLFromJson`; it might help to move the `isTtlCaseFieldPresent` inside `getTTLFromJson`.  Will probably simplify L125 to 137

You might also want to consider if `getTTLFromJson` should actually return `Optional<TTL>` to get rid of all the null checks